annotation/tests/test_annotations.py

```diff
@@ -107,6 +112,40 @@ def annotation_manifest():
     }
 
 
+@pytest.fixture
+def annotation_doc_for_save():
```

Comment:
If you're using this function in only one place, there's no need to create a fixture.

Reply:
It is used within another function as a parameter. I will leave it as a fixture.